components/services/markdown/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+const STATUS_OK = 200
+
+class ServiceMarkdown extends Component {
+  state = {html: ''}
+  async componentDidMount() {
+    const marked = await import('marked')
+    const req = new window.XMLHttpRequest()
+    req.open('GET', this.props.src, false)
+    req.send(null)
+    if (req.status === STATUS_OK) {
+      this.setState({html: marked(req.responseText)})
```

Comment:
seems that use marked as we use here is causing problems on their build form. Maybe we must change it for marked.default

Reply:
What version of sui-bundler are you using in order to get that error?